content/docs/get-started/quickstart.mdx

```diff
@@ -52,7 +52,7 @@ Paste the configuration in a `compose.yaml` file:
 ```yaml title="compose.yaml" {8,15} showLineNumbers
```

Comment:
Unrelated to this change, but the idea that you can copy the code here is wrong. If you click Finish before copying the docker configuration, you need to rotate your Zero Token to get another.